pkgs/applications/editors/jupyter-kernels/clojupyter/update.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/sh
+
+### To update clj2nix
+# $ nix-prefetch-github hlolli clj2nix
+
+nix-shell  --run "clj2nix deps.edn deps.nix" -E '
+with import <nixpkgs> { };
```

Comment:
Use a relative path.

Reply:
Sorry, I don't quite understand (this code was kindly provided by @bendlas) -- relative path to what?

Reply:
I think the idea is smth like `../../../../default.nix`, instead of `<nixpkgs>`. The reason I left `<nixpkgs>` was, that the script could be called from another working dir, and I didn't want to mess with bash-isms to get the script directory.

Also, I thought building the updater from whatever `<nixpkgs>` is current, may even be more appropriate, than building the current system to build the updater in this case. The intended audience for this script are only devs, hacking on nixpkgs.

OTOH, I only hacked this up as a proposal. No qualms about having it another way. I was actually thinking, that `clj2nix` should probably be packaged with nixpkgs, if we were to do this approach.

Reply:
It should use the current checkout because it is not guaranteed that the nixpkgs channel exists.

Reply:
I just changed it to a relative path and confirmed it works, although I tend to agree with @bendlas that it makes more sense to use `<nixpkgs>` since this update script is only for maintainers, and this relative path makes it more fragile to moving the file around etc.

Either way, this PR is ready for another look!

Reply:
> this relative path makes it more fragile to moving the file around etc.

The file usually does not move around and when you use the nix-channel you can easily bypass local changes and wonder why they are not used in the updater.